atc/api/handler.go

```diff
@@ -125,18 +125,22 @@ func NewHandler(
 
```

Comment:
I don't know why I can't put a comment on the line but I think we should probably not allow setting a pipeline if it is archived. Same with the `atc.CreateBuild`!

Reply:
Setting a pipeline is how you unarchive a pipeline

Reply:
`CreateBuild` is not pipeline scoped, it's team scoped. Used by one-off builds `fly execute` I think

Reply:
Ohh I didn't know you could unarchive a pipeline, okay that makes sense! And the `CreateBuild` makes sense too, that is fly execute right?